config/routes.rb

```diff
@@ -62,4 +62,10 @@
   mount RegistrationSharing::Engine, at: '/api/regsharing' if defined?(RegistrationSharing::Engine)
   mount InstanceVerification::Engine, at: '/api/instance' if defined?(InstanceVerification::Engine)
   mount SccSumaApi::Engine, at: '/api/scc' if defined?(SccSumaApi::Engine)
+
+  get '/connect/organizations/products/unscoped', to: redirect(path: '/api/scc/unscoped-products')
```

Comment:
Is it possible to move all of this  and the `mount`s above into a `if defined?(SccSumaApi::Engine)` block?